src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -747,4 +747,9 @@ Valid from:</comment>
   <data name="NuGetLicenseExpression_LicenseInvalidCharacters" xml:space="preserve">
     <value>The license identifier '{0}' contains invalid characters.</value>
   </data>
+  <data name="ConflictingAllowUntrustedRoot" xml:space="preserve">
+    <value>There are two certificates with conflicting allowUntrustedRoot in the computed settings. AllowUntrustedRoot is going to be set to false. Certificate: {0}-{1}</value>
```

Comment:
nit: I get it's the beginning of a sentence but maybe the AllowUntrustedRoot casing should be consistent. 

The real feedbackh ere is to add a comment specifying that allowUntrustedRoot should not be localized.

Reply:
I used that casing since `allowUntrustedRoot` uses that casing in the config file. I will add the comment to not localize it. Good catch!